### ISSUE TITLE
index_domain.pl - file documentation

### ISSUE THREAD

**Issue by holtzermann17** [dginev/nnexus#53]
https://github.com/dginev/nnexus/blob/master/util/index_domain.pl#L5-L7 doesn't seem to be correct.

> This script starts with a fresh NNexus SQLite database and performs an indexing pass over all defined Index Templates currently: PlanetMath, Wikipedia, DLMF and Mathworld

Actually, it looks like it just indexes one domain.

**Comment by dginev** [comment]
True. It started as traversing all, which then turned out too complex to do in a single process, so I simplified it to get the domain as a command-line argument, as in:
```
perl index_domain.pl Wikipedia
```
